### PR TITLE
Fixes for firewall

### PR DESCRIPTION
**Pull request by ypise**
Signed-off-by: Yashodhan Pise <yashodhan.pise@seagate.com>

### CODE REVIEW

File: pillar/components/firewall.sls [hunk 1]
```diff
@@ -10,9 +10,9 @@ firewall:
     services:
       - ssh
       - ftp
-      {%- if salt['cmd.run']('rpm -qa glusterfs-server') %}
+      {#%- if salt['cmd.run']('rpm -qa glusterfs-server') %#}
```
Comment (ypise) [inline pillar/components/firewall.sls:13]: This is probably a reason for the cache info output during destroy.

Comment (83bhp) [reply pillar/components/firewall.sls:13]: will this skip the glusterfs teardown?

Comment (ypise) [reply pillar/components/firewall.sls:13]: It won't skip teardown. This is just for firewall port for GlusterFS

**Comment by ypise** [comment]
This change is unnecessary.

**Comment by ypise** [comment]
This change is unnecessary.

**Pull request closed without merging**